src/test/java/org/opentripplanner/smoketest/SmokeTest.java

```diff
@@ -0,0 +1,152 @@
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.DeserializationContext;
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.JsonDeserializer;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.module.SimpleModule;
+import com.fasterxml.jackson.module.paramnames.ParameterNamesModule;
+import java.io.IOException;
+import java.net.URI;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse.BodyHandlers;
+import java.time.DayOfWeek;
+import java.time.LocalDate;
+import java.time.temporal.TemporalAdjusters;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.stream.Collectors;
+import org.opentripplanner.api.json.JSONObjectMapperProvider;
+import org.opentripplanner.api.model.ApiItinerary;
+import org.opentripplanner.api.resource.TripPlannerResponse;
+import org.opentripplanner.routing.core.Fare;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * This is both a utility class and a category to select or deselect smoke tests during test
+ * execution.
+ * <p>
+ * By default, the smoke tests are not run when you execute `mvn test`.
+ * <p>
+ * If you want run them use the following command: `mvn test -P smoke-tests`
+ */
+public class SmokeTest {
+
+    static final Logger LOG = LoggerFactory.getLogger(SmokeTest.class);
+    static HttpClient client = HttpClient.newHttpClient();
+    static final ObjectMapper mapper;
+
+    /**
+     * The Fare class is a little hard to deserialize so we have a custom deserializer as we don't
+     * run any assertions against the fares. (That is done during unit tests.)
+     */
+    static class FareDeserializer extends JsonDeserializer<Fare> {
+
+        @Override
+        public Fare deserialize(
+                JsonParser jsonParser, DeserializationContext deserializationContext
+        ) throws IOException, JsonProcessingException {
+            return null;
+        }
+    }
+
+    static {
+        var provider = new JSONObjectMapperProvider();
+
+        SimpleModule module = new SimpleModule("SmokeTests");
+        module.addDeserializer(Fare.class, new FareDeserializer());
+
+        mapper = provider.getContext(null);
+        mapper.configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false);
+        mapper.registerModule(new ParameterNamesModule(JsonCreator.Mode.PROPERTIES));
+        mapper.registerModule(module);
+    }
+
+    /**
+     * In order to have somewhat predictable results we get the route for the next Monday.
+     *
+     * FIXME: In the tech call on 2021-12-14 we said that we want to use the next business day, but
+     * as of 2021-12-15 this is not possible because the trips in the MARTA feed available to download
+     * start on 2021-12-18. After that date we can actually implement this logic.
```

Comment:
I think this part of the comment can be removed now. Or maybe change it to say we could expect failures in the days leading to a planned service change as the corresponding GTFS would not be available yet.